<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 46, 46);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TitleBackround">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>65</y>
      <width>511</width>
      <height>141</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>86</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>L GICA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TitleForground">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>86</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(126, 217, 87);</string>
    </property>
    <property name="text">
     <string>O</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="playTitleButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Images.qrc">
      <normaloff>:/OpenScreen/PlayPressed.png</normaloff>
      <normalon>:/OpenScreen/Play.png</normalon>:/OpenScreen/PlayPressed.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="learnTitleButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Images.qrc">
      <normaloff>:/OpenScreen/library.png</normaloff>:/OpenScreen/library.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="helpTitleButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Images.qrc">
      <normaloff>:/OpenScreen/info.png</normaloff>:/OpenScreen/info.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <zorder>TitleBackround</zorder>
   <zorder>playTitleButton</zorder>
   <zorder>learnTitleButton</zorder>
   <zorder>helpTitleButton</zorder>
   <zorder>TitleForground</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReset_Title">
    <property name="title">
     <string>Screen</string>
    </property>
    <addaction name="actiontitle"/>
   </widget>
   <addaction name="menuReset_Title"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiontitle">
   <property name="text">
    <string>Title</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
